SpatialGDK/Source/SpatialGDK/Public/Utils/EntityPool.h

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+
+#include "EngineClasses/SpatialNetDriver.h"
+#include "Utils/SchemaUtils.h"
+
+#include <WorkerSDK/improbable/c_schema.h>
+#include <WorkerSDK/improbable/c_worker.h>
+
+#include "EntityPool.generated.h"
+
+struct EntityRange
+{
+	Worker_EntityId CurrentEntityId;
+	Worker_EntityId LastEntityId;
+	bool bExpired;
+	// Used to identify an entity range when it has expired.
```

Comment:
Move comment to be beside or above the bool

Reply:
This is a comment for the `EntityRangeId`.